cli-tests/src/upload.rs

```diff
@@ -51,17 +48,27 @@ async fn upload_bundle() {
     assert_eq!(requests.len(), 5);
     let mut requests_iter = requests.into_iter();
 
-    assert_eq!(
-        requests_iter.next().unwrap(),
-        RequestPayload::GetQuarantineBulkTestStatus(GetQuarantineBulkTestStatusRequest {
-            repo: Repo {
-                host: String::from("github.com"),
-                owner: String::from("trunk-io"),
-                name: String::from("analytics-cli"),
-            },
-            org_url_slug: String::from("test-org"),
-        })
-    );
+    let quarantine_request = requests_iter.next().unwrap();
+    if let RequestPayload::GetQuarantineBulkTestStatus(req) = quarantine_request {
```

Comment:
nit: use `assert_matches!()` for this. There's an example below on how to use it